### ISSUE TITLE
[suggestion] setting to manually set fps cap

### ISSUE THREAD

**Issue by amplified1** [tsunamayo/Starship-EVO#3333]
Although the v-sync is good the average fps of this game for me is less that 144 fps and my GPU constantly runs at 100 percent usage with this open. Being able to cap it at like 60fps would be helpful to circumvent this until more graphics optimisations are made.

**Comment by tsunamayo** [comment]
Yes I agree, I will add ratio that will be better for high refresh rate monitor.
